[5.x] Improve Search result message

chmst opened the pull request:
Pull Request für Issue # .

### Zusammenfassung der Änderungen
siehe https://github.com/joomla/joomla-cms/issues/45182#issuecomment-2743433160

Die Ergebnisse der erweiterten Suche werden falsch angekündigt. Eine 1:1 Übersetzung ist nicht möglich. 
Dies ist ein Vorschlag für eine Übersetzung, ohne dass am Core was geändert werden muss.

Mit der Änderung würde die Ausgabe so aussehen: 

![grafik](https://github.com/user-attachments/assets/0b1bbd3c-0e0e-4839-a34b-b009570242d2)

Achtung: Es gibt viele verschiedene Testfälle.

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden
Searchindex modul, Erweiterte Suche.

chmst committed:
Improve Search result message

tecpromotion committed:
Merge branch '5.3-dev' into patch-2

tecpromotion commented:
Danke @chmst 
Ich schaue es mir die Tage an. Rebase auf 5.3-dev habe schon gemacht, es wird keine Version mehr für 5.2 geben.

chmst commented:
"Für den Suchbegriff" ist nicht ganz korrekt, es sind ja zwei Suchbegriffe. 

Offen ob man sagt "Für die Suche nach.. "

This pull request includes changes to the German language file `com_finder.ini` to improve the clarity and consistency of the search result messages. The most important changes include the removal of unnecessary commas and the addition of parentheses for better readability.

Improvements to search result messages:

* Removed unnecessary commas in `COM_FINDER_QUERY_RESULTS`, `COM_FINDER_QUERY_RESULTS_0`, and `COM_FINDER_QUERY_RESULTS_1` to improve sentence flow.
* Added parentheses around terms in `COM_FINDER_QUERY_TOKEN_EXCLUDED`, `COM_FINDER_QUERY_TOKEN_OPTIONAL`, and `COM_FINDER_QUERY_TOKEN_REQUIRED` for better readability and consistency.

tecpromotion commented:
1. Suche nach "doof **und** clever"

![Bildschirmfoto 2025-03-22 um 10 04 52](https://github.com/user-attachments/assets/0e997c29-1b35-40ff-a47c-fb24673fda5c)

2. Suche nach "doof" 

![Bildschirmfoto 2025-03-22 um 10 04 32](https://github.com/user-attachments/assets/b405afd3-c188-46d4-b7b6-366aba02f3a8)

3. Suche nach "clever"

![Bildschirmfoto 2025-03-22 um 10 04 13](https://github.com/user-attachments/assets/3ac784b3-ad94-44b1-bef8-b0bb01a4137e)

4. Suche nach "doof **oder** clever"

![Bildschirmfoto 2025-03-22 um 10 03 36](https://github.com/user-attachments/assets/810b439d-33bb-4884-9a45-77e4131af562)

5. Suche nach "dumm"

![Bildschirmfoto 2025-03-22 um 10 08 32](https://github.com/user-attachments/assets/a88e29c1-d274-41b2-8730-22db3d300437)

6. Suche nach "doof **nicht** clever"

![Bildschirmfoto 2025-03-22 um 10 08 21](https://github.com/user-attachments/assets/b3d8aa03-0c11-47b4-a5a7-770563c22541)

tecpromotion reviewed (approved):
Danke @chmst
Ich hab mir das angesehen und das sieht OK aus.

tecpromotion commented:
@heelc29 oder @zero-24 noch Anmerkungen?

@crimle gerne auch mal testen.

zero-24 commented:
> @heelc29 oder @zero-24 noch Anmerkungen?
> 
> @crimle gerne auch mal testen.

Entschuldige die Nachricht ist mir unter gegangen. Passt für mich vielen dank euch 👍
